JAICore/jaicore-ml/src/main/java/jaicore/ml/scikitwrapper/AProcessListener.java

```diff
@@ -0,0 +1,31 @@
+package jaicore.ml.scikitwrapper;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public abstract class AProcessListener implements IProcessListener {
```

Comment:
Please add brief JavaDoc description for this class is good for. (also to the methods)

Reply:
Done